.gitmodules

```diff
@@ -75,3 +75,9 @@
 [submodule "src/jaegertracing/opentelemetry-cpp"]
 	path = src/jaegertracing/opentelemetry-cpp
 	url = https://github.com/open-telemetry/opentelemetry-cpp.git
+[submodule "src/qatlib"]
```

Comment:
IMHO, it's not be appropriate to always download the latest version of the qatlib and QATzip repositories. Maybe branch [23.02.0](https://github.com/intel/qatlib/tree/23.02.0) for qatlib, branch [v1.1.2](https://github.com/intel/QATzip/tree/v1.1.2) for QATzip.

Reply:
the submodules point to refs https://github.com/intel/qatlib/tree/142e305970ec66a860945d20bb7c330f99ed900b and https://github.com/intel/qatzip/tree/fdee557b5bb640827758f121102dcf3583292b7a, which correspond to the latest qatlib 23.11.0 and qatzip 1.1.2 releases

Reply:
got it. Learned how to use git submodule to add submodules